pages/_app.tsx

```diff
@@ -22,6 +22,10 @@ const Link: React.FunctionComponent<LinkComponentTypeProps> = props => {
   const { href: originalHref } = props;
   const { href, as } = urlsForRouter(originalHref);
 
+  if (!originalHref.startsWith("/") || originalHref.startsWith("//")) {
```

Comment:
This is confusingly expresses isn't it? Maybe it would be better to reverse the logic to wrap the internal links and fall through to a plain `a`?